### PR TITLE
Added In Assignment 07 & 08

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

File: Ashwin/JavaScript/08_Assignment/08_Assignment_1.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                        subjectMarks.push(marks);
+                    }
+                    subject--;
+                }
+                subjectMarks.forEach((value, index) => {
+                    result.innerHTML += `<br> Subject: <b>${index + 1}</b> = Marks: <b>${value}</b>`;
+                });
+            }
+        </script>
+        <title>Student Marks</title>
```
Comment (nileshp1122) [inline Ashwin/JavaScript/08_Assignment/08_Assignment_1.html]: title required before script tag

File: Ashwin/JavaScript/08_Assignment/08_Assignment_2.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                background: url(plain.jpg);
+                margin-top: 150px;
+            }
+            p {
+                text-align: center;
+                font-size: 30px;
+                color: blueviolet;
+            }
+        </style>
+        <title>Q1</title>
```
Comment (nileshp1122) [inline Ashwin/JavaScript/08_Assignment/08_Assignment_2.html]: proper title and title before script tag

File: Ashwin/JavaScript/08_Assignment/08_Assignment_3.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                    result.innerHTML += "<br><br><b>Total Marks After Applying Sort</b>";
+                    subjectMarks.forEach((value, index) => {
+                        result.innerHTML += `<br> Subject: <b>${index + 1}</b> == Marks: <b>${value}</b>`;
+                    });
+                } else {
+                    result.innerHTML += "<br><b>Total Marks After Applying Sort</b>";
+                }
+            }
+        </script>
+        <title>Marks Sorting Using Different Technique</title>
```
Comment (nileshp1122) [inline Ashwin/JavaScript/08_Assignment/08_Assignment_3.html]: title before script, style, link tag

File: Ashwin/JavaScript/08_Assignment/08_Assignment_4.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+                background-size: cover;
+                margin-top: 100px;
+            }
+            p {
+                text-align: center;
+                font-size: 30px;
+                color: blueviolet;
+            }
+        </style>
+        <title>Merge Sort</title>
```
Comment (nileshp1122) [inline Ashwin/JavaScript/08_Assignment/08_Assignment_4.html]: title before script, style, link tag

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**